crates/common/download/src/download.rs

```diff
@@ -415,6 +415,7 @@ async fn save_chunks_to_file_at(
     while let Some(bytes) = response.chunk().await? {
         writer.write_all(&bytes)?;
     }
+    writer.flush()?;
```

Comment:
note: this is an example where there's no functional change, because [`flush` on a `File` is a noop](https://doc.rust-lang.org/std/fs/struct.File.html#platform-specific-behavior-14), but still it's probably good to have for consistency or if somebody added a buffer and forgot to add a `flush` call.

If we actually want to ensure that all writes reach the file, I think `sync_data` or `sync_all` might be better here.